keyboards/handwired/stream_cheap/2x5/rules.mk

```diff
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+
+
+AUDIO_ENABLE = no
+BACKLIGHT_ENABLE = no   # Enable keyboard backlight functionality
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+COMMAND_ENABLE = no     # Commands for debug and configuration
+CONSOLE_ENABLE= no	# Console for debug
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+MOUSEKEY_ENABLE = yes	# Mouse keys
+NKRO_ENABLE = yes	# USB Nkey Rollover -
+
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+UNICODE_ENABLE = yes
+TAP_DANCE_ENABLE = no
```

Comment:
```suggestion
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```
```